src/Homebrew.jl

```diff
@@ -16,7 +16,11 @@ const brew_prefix = abspath(joinpath(dirname(@__FILE__),"..","deps", "usr"))
 const brew_exe = joinpath(brew_prefix,"bin","brew")
 const tappath = joinpath(brew_prefix,"Library","Taps","staticfloat","homebrew-juliadeps")
 
-const BREW_URL = "https://github.com/Homebrew/homebrew.git"
+if VERSION < v"0.5.0-dev+522"
+    const BREW_URL = "https://github.com/Homebrew/homebrew.git"
+else
+    const BREW_URL = "https://github.com/Homebrew/brew.git"
+end
```

Comment:
ah was wondering how hard it is to do clean vm's.

also the migration path from existing clones of the old formulas-and-pkgmanager repo to the new separate pkgmanager vs formulas tap repos. we'd have to reimplement their logic to do that properly. maybe we shell out to brew update only once if we detect the old remote?